src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationResidualVsTimeChartPresenter.cs

```diff
@@ -120,13 +120,18 @@ private DataRepository createScatterDataRepository(string id, string repositoryN
       {
          var dataRepository = createEmptyRepository(id, repositoryName, "Values");
          var scatterColumn = dataRepository.FirstDataColumn();
-         scatterColumn.QuantityInfo.Path = outputResidual.FullOutputPath.ToPathArray();
+         var outputPath = new List<string>(outputResidual.FullOutputPath.ToPathArray());
+         //need to create a unique path containing the observed data name. Since we want to keep the entity name and simulation name, we have to insert before the name
+         if (outputPath.Count > 0)
```

Comment:
This is the actual fix. Problem was that I had many curves with the same output. When applying the template, they would either all disappear from legend or all be added to the legend depending on some order. Now the full path is unique and the issue is solved